src/storage/persistent/webext.ts

```diff
@@ -146,7 +146,7 @@ class WebExtStore implements Store {
       );
       return this.store.set(query);
     } catch(e) {
-      console.warn(e.message, "Ignoring");
+      console.warn((e as Error).message, "Ignoring");
```

Comment:
```suggestion
      console.warn("Ignoring key", e);
```

This is another option. The whole exception object will be accessible in the console.